source/reference/minio-mc-admin/mc-admin-logs.rst

```diff
@@ -0,0 +1,116 @@
+=================
+``mc admin logs``
+=================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support logs show
+.. mc:: mc admin logs
+
+.. include:: /includes/common-mc-support.rst
+   :start-after: start-minio-only
+   :end-before: end-minio-only
+
+.. versionchanged:: RELEASE.2022-12-02T23-48-47Z
+
+   ``mc support logs`` moved to ``mc admin logs`` and provide a simpler command interface for displaying console logs for the MinIO deployment.
```

Comment:
```suggestion
   ``mc support logs`` moved to ``mc admin logs`` and provides a simpler command interface for displaying server logs for the MinIO deployment.
```

I think? I prefer "server logs" because it's not ambiguous with the actual Console.

Reply:
Could also state that the output is similar to what is available via `journalctl -uf minio` for systemd-controlled MinIO deployments.